www/cgi-bin/clear

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+ > ${NVOC}/7_temp_alertlog
+ > ${NVOC}/7_wdog_alertlog
```

Comment:
These 2 files are not used any more, the 2 warning files for temp control and watchdog are:
/home/m1/5_watchdoglog
/home/m1/6_autotemplog

Reply:
What id I completely remove the that "clear" file? Is it still used somewhere?